### ISSUE TITLE
[Bug]: not able to set default image in theme config

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#5115]
### What e107 version are you using?

Latest Github version (just updated)

### Bug description

I need the default image for theme config  field

if I use this:
```
	<themePrefs>
		<artemis name="frontimage">{THEME}soft-ui-design-system/images/curved.jpg</artemis>
	</themePrefs>
```
it is trying to display:

`<img class="image-selector img-responsive img-fluid" src="/e107233/e107_themes/bootstrap3/soft-ui-design-system/images/curved.jpg" alt="curved.jpg" width="206" height="190">`

 

### How to reproduce

No idea :)  Maybe install Artemis theme?  

### Expected behavior

Use a frontend theme path.

There may be another shortcode for this.   But theme name shouldn't be part of it. 

Thanks

### What browser(s) are you seeing the problem on?

Chrome / Brave

### PHP Version

8.1

**Comment by Moc** [comment]
Shouldn't it be `<pref name="...` rather than `<artemis name="...`?

```xml
<themePrefs>
	<pref name="frontimage">{THEME}soft-ui-design-system/images/curved.jpg</artemis>
</themePrefs>
```

**Comment by Jimmi08** [comment]
> Shouldn't it be `<pref name="...` rather than `<artemis name="...`?
> 
> ```
> <themePrefs>
> 	<pref name="frontimage">{THEME}soft-ui-design-system/images/curved.jpg</artemis>
> </themePrefs>
> ```

No, from install.xml it has different syntax., but this one works (only path is wrong).  When you export theme prefs from Database/tools and you want to import them back, you need to use different syntax then in theme.xml.  
Weird, I admit.
By the way, it didn't work from theme.xml, probably because shortcode in value.
```
	<themePrefs>
		<pref name='frontimage'>{e_THEME}artemis/images/curved.jpg</pref>
	</themePrefs>
```

Finally, I used {e_THEME} in install.xml, but still...  but the problem with using frontend theme stuff in the admin area stays.
